services/client/src/styles/_detail.scss

```
@@ -10,60 +16,137 @@
 .detail-title {
-    margin: 0 0 0.75rem;
-    font-size: 1.25rem;
-    color: $color-primary;
-    word-break: break-all;
+    margin: 0;
+    font-size: 1.35rem;
+    font-weight: 700;
+    letter-spacing: -0.01em;
+    color: $color-text;
+    line-height: 1.3;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)